app/api.py

```
@@ -2,16 +1,26 @@
+# app/api.py
+from fastapi import APIRouter, Request, HTTPException, status, Depends
```

Ruff (0.8.2)

`fastapi.status` imported but unused

Remove unused import: `fastapi.status`

(F401)